### ISSUE TITLE
Allow group admins to more easily view more or all group members

### ISSUE THREAD

**Issue by bree-z** [cuny-academic-commons/commons-in-a-box#228]
Hi Boone, 

We talked about a way to allow group admins to view more than 12 members at a time, and you suggested that using the new BuddyPress interface for managing membership would be the best way to do this, but it would need design. 

@jennaspevack would like to see what is involved, but needs a place to see what this new interface looks like so she can figure out whether it's doable for her right now.

Can you point her to somewhere that she can get started?

And if this doesn't work out, we'd like to take an intermediate step of changing the default view when managing group membership to display 48 members rather than 12.

Thanks!

**Comment by boonebgorges** [comment]
I spent a few minutes to see if it was straightforward to integrate the BP interface into `openlab-theme`, at least in a rough way. I was able to make it work, but it was completely unstyled, so it's going to take a lot of work to make it usable.

For Jenna to be able to test this interface, I'd have to set up a separate, vanilla BP sandbox. This is not trivial for me to do at this time. As a stopgap, I created a short gif that demonstrates how the functionality works in BP out-of-the-box.
![Peek 2020-05-28 21-58](https://user-images.githubusercontent.com/246627/83216428-d73eb500-a12e-11ea-80de-dbca64fbc6f9.gif)

Maybe this will be enough of a teaser to decide whether it's a good starting place for OL/CBOX integration.

In the meantime, I've increased the number of users shown on `manage-members` from 15 to 48.
